app/assets/stylesheets/application.css.scss

```diff
@@ -11,6 +11,7 @@
  * It is generally better to create a new file per style scope.
  *
  *= require font-awesome
+ *= require variables
```

Comment:
The files that are `//= require`d are compile separately and then concatenated into the final CSS bundle so your Sass files will not be able to access the variables defined in `_variables.scss`.

Don't use `= require` in Sass files, use `@import` instead, e.g.:

```scss
@import 'variables';
```